tests/s3-test.js

```diff
@@ -0,0 +1,168 @@
+'use strict';
+
+const assert = require('assert');
+const sandbox = require('sinon').createSandbox();
+
+const s3Wrapper = require('../lib/s3Wrapper');
+
+const S3 = require('./../lib/s3');
+
+describe('S3', () => {
+
+	afterEach(() => {
+		sandbox.restore();
+	});
+
+	const s3Params = {
+		Body: '<Binary String>',
+		Bucket: 'examplebucket',
+		Key: 'objectkey'
+	};
+
+	context('getObject', () => {
+
+		it('should return a promise with the same params calling to getObject method', async () => {
```

Comment:
```suggestion
		it('Should return a promise with the same params calling to getObject method', async () => {
```

Y mayus a todos los demás _should_'s

Reply:
PD: no, no debería devolver los params, sino que debería devolver lo mismo que devuelve el SDK de AWS.